pkg/scheduler/extender/extender.go

```diff
@@ -253,15 +255,20 @@ func (e *Extender) gatherCapacityRequestsByProvisioner(ctx context.Context, pod
 				return nil, baseerr.ErrorNotFound
 			}
 
-			ll.Debugf("PVC details: %s", pvc.String())
+			ll.Debugf("PVC %s status: %+v", pvc.Name, pvc.Status)
+			ll.Debugf("PVC %s spec: %+v", pvc.Name, pvc.Spec)
```

Comment:
Won't it print SC name from spec as well?

Reply:
It is a pointer - `StorageClassName:0xc000434670`